tests/st/utils/utils.py

```diff
@@ -147,6 +147,22 @@ def assert_no_error(self, text=None):
         if text:
             self.assert_output_contains(text)
 
+    def check_reports(self, reports):
+        """
+        Checking for unexpected reports.
+        """
+        unexpected_report = False
+        for report in reports:
```

Comment:
I think it would be better to output *all* of the report files if there is any unexpected file, rather than only outputting the unexpected file.